reporter/src/main/kotlin/model/EvaluatedModelMapper.kt

```diff
@@ -503,8 +501,9 @@ internal class EvaluatedModelMapper(private val input: ReporterInput) {
         scanResult: ScanResult,
         evaluatedScanResult: EvaluatedScanResult,
         findings: MutableList<EvaluatedFinding>,
-        licenseFindingCurations: Collection<LicenseFindingCuration>
+        id: Identifier
```

Comment:
Can you make this the first parameter of the function? The `id` is always the first parameter throughout ORT source code, and it makes sense from a hierarchical point of view, because the data below relates to that id.

Reply:
Done.